index.js

```diff
@@ -1,2 +1,2 @@
-// eslint-disable-next-line import/no-commonjs
+// eslint-disable-next-line import/no-commonjs, import/unambiguous
```

Comment:
I think you should put these rules in the `.eslintrc` file instead of `index.js`